models/cohortdefinition.go

```diff
@@ -35,11 +36,14 @@ type CohortDefinitionStats struct {
 }
 
 func (h CohortDefinition) GetCohortDefinitionById(id int) (*CohortDefinition, error) {
+	atlasDb := db.GetAtlasDB()
 	db2 := db.GetAtlasDB().Db
 	var cohortDefinition *CohortDefinition
 	query := db2.Model(&CohortDefinition{}).
-		Select("id, name, description").
-		Where("id = ?", id)
+		Select("cohort_definition.id, cohort_definition.name, cohort_definition.description, cohort_definition_details.expression").
+		Where("cohort_definition.id = ?", id).
+		Joins("LEFT JOIN " + atlasDb.Schema + ".cohort_definition_details ON cohort_definition.id = cohort_definition_details.id")
```

Comment:
I think INNER JOIN reflects the expectation better. There should be no cohortdefinition without details...